nav2_util/include/nav2_util/service_client.hpp

```diff
@@ -48,6 +49,18 @@ class ServiceClient
       service_name,
       rclcpp::SystemDefaultsQoS(),
       callback_group_);
+    rcl_service_introspection_state_t introspection_state = RCL_SERVICE_INTROSPECTION_OFF;
+
```

Comment:
hHis is where you can declare and get the service introspection value internally to the class!